packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -127,6 +145,16 @@ contract Deploy is Deployer {
         }
     }
 
+    function deploySuperchainConfigProxy() public broadcast returns (address addr_) {
+        Proxy proxy = new Proxy({
+            _admin: msg.sender
```

Comment:
It looks like the deployer is the admin of the proxy at the end of the script, we want the proxy admin to be the admin?

Reply:
Fixed